ethereum-contracts/README.md

```diff
@@ -1,72 +1,61 @@
-# Testing contracts
+# Unidirectional Peggy Project Specification
 
-## Installing dependencies
+## Project Summary
+Unidirectional Peggy is the starting point for cross chain value transfers from Ethereum to Cosmos as part of the Ethereum Cosmos Bridge project. The smart contract system accepts incoming transfers of Ethereum and ERC20 tokens, locking them while the transaction is validated and equitable funds issued to the intended recipient on Cosmos.
 
-  - Truffle: `npm install -g truffle` for Mac or go to the install
-  - Testrpc: `npm i -g ethereumjs-testrpc`
-  - Ganache: Download file [here](http://truffleframework.com/ganache/)
+## Project Background
+We are hoping to create a closed system for intra network transfers of cryptocurrency between blockchains, spearheaded by a proof-of-concept which enables secured transactions between Ethereum and Cosmos.
 
-## Instructions
+## Smart Contract Scope
 
-You can test the contracts using Truffle Testrpc in the console or Ganache UI.
+### Goals of the Smart Contracts
+1. Securely implement core functionality of the system such as asset locking and event emission without endangering any user funds. As such, this prototype does not permanently lock value and allows the original sender full access to their funds at any time.
+2. Interface with the Relayer service, which is used by validators to listen for contract events which are signed and submitted to the Cosmos network as proof of transaction.
```

Comment:
Is the "Relayer" a service or a module (or both)? It's called both in different places in this file.

Reply:
The Relayer is a service, apologies for the confusion.